### PR TITLE
Add ruizhg to typescript readability

### PR DESCRIPTION
**Pull request by ruizhg**

### CODE REVIEW

File: terraform/typescript.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 chloechien: 'maintainer'
 verbanicm: 'maintainer'
 mattrandallbecker: 'maintainer'
+ruizhg: 'maintainer'
```
Comment (verbanicm) [inline terraform/typescript.yaml:5]: Can you move this below line 8 and change to `member` instead of `maintainer`?


```
ruizhg: 'member'
```

**Review by verbanicm — CHANGES_REQUESTED** [review]
See comment

File: terraform/typescript.yaml (hunk 1 above)
Comment (ruizhg) [reply terraform/typescript.yaml]: Updated

**Review by verbanicm — DISMISSED** [review]

File: terraform/typescript.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ mattrandallbecker: 'maintainer'
 
 # Members - to add a new member, add their GitHub username and propose a PR with
 # them as a reviewer.
+ruizhg: 'member'
 #
```
Comment (chloechien) [inline terraform/typescript.yaml:9]: Feel free to remove this comment block

**Review by chloechien — DISMISSED** [review]

File: terraform/typescript.yaml (hunk 2 above)
Comment (ruizhg) [reply terraform/typescript.yaml]: removed. But I lose the approvals.

**Review by verbanicm — DISMISSED** [review]

**Review by chloechien — APPROVED** [review]

File: terraform/typescript.yaml [hunk 3]
```diff
@@ -5,6 +5,4 @@ mattrandallbecker: 'maintainer'
 
 # Members - to add a new member, add their GitHub username and propose a PR with
 # them as a reviewer.
-#
```
Comment (sethvargo) [inline terraform/typescript.yaml:8]: Please don't delete these lines.

Comment (ruizhg) [reply terraform/typescript.yaml:8]: Added the comment block back.

**Review by chloechien — APPROVED** [review]

**Review by verbanicm — APPROVED** [review]

**Pull request merged**